contribution_guidelines.rst

```diff
@@ -0,0 +1,93 @@
+
+This document provides a set of guidelines and information for contributing to icepyx.
+
+
+Ways to Contribute
+------------------
+
+- Share your use cases and examples (as Jupyter Notebooks, scripts, etc.)
+- Submit bug reports and feature requests
+- Write code for everyone to use
+- Fix typos
+- Improve documentation and testing
+
+The best way to report a problem, request a feature, find out if others are working on a similar problem or application, or let us know you'd like to write some code is to find the *Issues* tab and check if your problem/suggestion has already been reported. If so, please provide any additional information in the ongoing discussion. Otherwise, feel free to create a new issue and submit your problem or suggestions.
+
+
+Reporting a Bug
+---------------
+Find the *Issues* tab at the top of GitHub repository and click *New Issue* button. Give your issue a clear title and describe the steps required to recreate it in as much detail as possible. If you can, include a small example that reproduces the error. More information and minimal examples will help us resolve issues faster.
+
+DevGoal: separate section for feature requests/suggestions? Should we provide issue/feature templates?
+
+
+Questions and Help
+------------------
+
+Please do not create issues to ask for help. A faster way to reach the community is through our Science/ICESat-2 subcategory on the `Pangeo discourse <https://discourse.pangeo.io/c/science/icesat-2/16>`_ page. We are excited to have you join an existing conversation or start a new post! Please note that a github login is required to post on the discourse page.
+
+Other Resources
+^^^^^^^^^^^^^^^
+- We are currently working on compiling a list of resources for working with ICESat-2 data (in the meantime, check out the `2019 ICESat-2 Hackweek repo<https://github.com/ICESAT-2HackWeek>`_ and `consider applying for this year's Hackweek<https://icesat-2hackweek.github.io/>`) 
+- A great set of `interactive tutorials<https://learngitbranching.js.org>`_ for learning and practicing using git
+- Let us know about the helpful tools you've found!
+
+
+Adding Examples
+---------------
+We are delighted you'd like to contribute your icepyx/ICESat-2 example! Examples may be in the form of executable scripts or interactive Jupyter Notebooks. Please make sure that each example has a descriptive name so someone not familiar with your project understands its general behavior. Fully working examples should be submitted using a pull request to the "examples" branch, following the steps outlined below for **Contributing Code**.
+
+Question - Do we want to strongly suggest all examples submitted be in python? Are they required to use icepyx, or just ICESat-2 data?
```

Comment:
I think in this repo we should limit it to icepyx. Having a matlab example using third party tools in this repo would be weird...

We could, if there's enough interest, start a companion repo for illustrative examples that use any other toolchain, in the top-level icesat2py org (though the 'py' is a bit of a hint that we might want to shed then :)

Reply:
I agree - I think a companion repo, should the need arise, is a great way to make sure anyone working with ICESat-2 data can contribute/participate. I've modified the text to just say icepyx examples.